### ISSUE TITLE
Link on Print Application Preview lacks visual indication that its a link

### ISSUE THREAD

**Issue by slowbot** [bloom-housing/bloom#3160]
As a user,  I expect links should have a visual indication of then color alone that they are clickable

- It looks like we lost link styles on this button at the foot of the print application preview at the end of the application

/applications/view

![image](https://user-images.githubusercontent.com/13915/198700440-ab5d4e06-c0f9-4ea4-9470-1302ca2654c1.png)

**Comment by emilyjablonski** [comment]
Fixed
